index.js

```diff
@@ -220,20 +220,22 @@ async function createNewExplanation(info) {
 
     // If the comic title is a number underneath the current comic number, do not create this redirect
     if (!/^\d+$/.test(comicTitle) || +comicTitle > comicNum) {
+      const comicTitleRedirect = `#REDIRECT [[${comicNum}: ${comicTitle}]]\n`;
       await bot.edit(
         comicTitle,
-        `#REDIRECT [[${comicNum}: ${comicTitle}]]\n`,
-        EDIT_SUMMARY,
+        content,
+        EDIT_SUMMARY + comicTitleRedirect,
```

Comment:
For these, could you use string templates rather than addition?
```js
`${EDIT_SUMMARY}${comicTitleRedirect}`
```

Reply:
Thanks. Applied the string template change in 740f5e9.